docs/operations/using-rolling-upgrades.md

```diff
@@ -0,0 +1,44 @@
+---
+linkTitle: Platform Upgrade
+---
+
+# Overview
+
+This guide will demonstrate how to enable the orchestrator's rolling upgrades using the [Upgrade Guard](/docs/concepts/rolling-upgrades/#upgrade-guard) and [Node Manager](/docs/concepts/rolling-upgrades/#node-manager). This feature helps to prevent your persistent storage volumes from becoming unhealthy during an orchestrator update.
+
+# Prerequisites
+
+> ⚠️ Make sure you have met the requirements of [configuring a Pod Disruption Budget (PDB)](https://kubernetes.io/docs/tasks/run-application/configure-pdb/).
+
+> ⚠️ If your volume does not have any replicas, the rolling upgrades feature will not start on any StorageOS node until you have one or more replicas on all your volumes.
+
+> ⚠️ This feature supports the following platforms: AWS EKS, Google Anthos, Google GKE, Microsoft Azure, Openshift and Rancher.
```

Comment:
`Google Anthos` no-one never tested it on Anthos, so
a) We have to start it immediately with automated tests
b) remove it from the list

Reply:
We can remove it from the list